app/chinese_family_name_dict.txt

```diff
@@ -0,0 +1,484 @@
+怀	huai 
```

Comment:
Can you add a comment where this data comes from? Maybe add comment lines starting with "#", and add a logic to skip lines starting with "#".